engine/src/main/scala/cromwell/server/CromwellShutdown.scala

```diff
@@ -217,5 +218,13 @@ object CromwellShutdown extends GracefulStopSupport {
       logger.info("Stream materializer shut down")
       Future.successful(Done)
     }
+
+    // 7) Close out the backend used for WDL HTTP import resolution
+    // http://sttp.readthedocs.io/en/latest/backends/start_stop.html
+    coordinatedShutdown.addTask(CoordinatedShutdown.PhaseBeforeActorSystemTerminate, "wdlHttpImportResolverBackend") { () =>
+      HttpResolver.closeBackendIfNecessary()
+      logger.info("WDL HTTP import resolver closed")
+      Future.successful(Done)
```

Comment:
Is the fact that this wants a `Future` back indicating that this should really be non-blocking, ie:
```scala
Future {
  HttpResolver.closeBackendIfNecessary()
  logger.info("WDL HTTP import resolver closed")
}
```

Reply:
The shutdown tasks seem to bucket into "takes a while" which is implemented like your example and "pretty fast" which do their work synchronously.

If closing the backend is a tossup, I will conservatively move it to async.